Source/JavaScriptCore/API/ObjCCallbackFunction.mm

```diff
@@ -727,13 +727,13 @@ JSObjectRef objCCallbackFunctionForMethod(JSContext *context, Class cls, Protoco
 {
     NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[NSMethodSignature signatureWithObjCTypes:types]];
     [invocation setSelector:sel];
-    if (!isInstanceMethod) {
-        [invocation setTarget:cls];
-        // We need to retain the target Class because m_invocation doesn't retain it by default (and we don't want it to).
-        // FIXME: What releases it?
-        CFRetain((__bridge CFTypeRef)cls);
-    }
-    return objCCallbackFunctionForInvocation(context, invocation, isInstanceMethod ? CallbackInstanceMethod : CallbackClassMethod, isInstanceMethod ? cls : nil, _protocol_getMethodTypeEncoding(protocol, sel, YES, isInstanceMethod));
+    if (isInstanceMethod)
+        return objCCallbackFunctionForInvocation(context, invocation, CallbackInstanceMethod, cls, _protocol_getMethodTypeEncoding(protocol, sel, YES, YES));
+
+    // We need to copy the target Class because m_invocation doesn't retain it by default (and we don't want it to).
+    [invocation setTarget:[cls copy]];
```

Comment:
This comment no longer makes when you replace the word "retain" with "copy".

What releases this copy we made of the class?